### PR TITLE
feat: map center button

### PR DESCRIPTION
**Pull request by luke-mcfarlane-rocketlab**
## JIRA Ticket

[BSS-289](https://jira.csiro.au/browse/BSS-289)

## Description

Added button to map component to move the map back to the initial location.

## Proposed Changes

- Created centre icon HTML element
- Created centre control button HTML element
- Added centre control to map component
- Added styling for map centre button in global App.css file

## How to Test

1. Activate "Campus Survey Demo" if it is not already activated
2. Open the survey "Campus Survey Demo"
3. Click the "+ Survey Area" button to add a new survey area
4. Click the "DRAW BOUNDING BOX AROUND SURVEY AREA"
5. Move the map
6. Test the centre button on the bottom left of the map screen

## Additional Information

HTML elements were used instead of React components as these are what OpenLayers expects for map components. See [OpenLayers v10.0.0 API - Class: Control](https://openlayers.org/en/latest/apidoc/module-ol_control_Control-Control.html)

## Checklist

- [x] I have confirmed all commits have been signed.
- [x] I have added JSDoc style comments to any new functions or classes.
- [x] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

**Review by stevecassidy — APPROVED** [review]
Looks good.

File: app/src/gui/components/map/center-control-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  path.setAttribute(
+    'd',
+    'M12 8c-2.21 0-4 1.79-4 4s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 6c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2zm8.94-3c-.46-4.17-3.77-7.48-7.94-7.94V1h-2v2.06C6.83 3.52 3.52 6.83 3.06 11H1v2h2.06c.46 4.17 3.77 7.48 7.94 7.94V23h2v-2.06c4.17-.46 7.48-3.77 7.94-7.94H23v-2h-2.06zM12 19c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7z'
+  );
+  path.setAttribute('fill', '#ffffff');
+
+  svg.appendChild(path);
+
+  return svg;
+};
```
Comment (PeterBaker0) [inline app/src/gui/components/map/center-control-icon.tsx:22]: Is this a maintainable way to create the icon? Can we load it from an svg file instead so it's a bit more visible what the icon is at a glance and IDEs can preview it etc.

**Review by PeterBaker0 — COMMENTED** [review]
Functionality looks good, I'm a little wary about writing raw svg into the react function. Is there an approach where we can still have control of the svg but load it from a file which can sit in the repo somewhere easily previewed?

File: app/src/gui/components/map/center-control-icon.tsx (hunk 1 above)
Comment (luke-mcfarlane-rocketlab) [reply app/src/gui/components/map/center-control-icon.tsx]: actioned - I have moved the svg into a separate file

**Pull request merged**